pkgs/development/python-modules/hikari/default.nix

```diff
@@ -0,0 +1,73 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pytestCheckHook
+, pytest-runner
+, aiohttp
+, attrs
+, multidict
+, colorlog
+, pynacl
+, git
+, pytest
+, pytest-cov
+, pytest-randomly
+, pytest-asyncio
+, mock
+  # Opts
+, enableRestServer ? false
+}:
+
+buildPythonPackage rec {
+  pname = "hikari";
+  version = "2.0.0.dev119";
+
+  src = fetchFromGitHub {
+    owner = "hikari-py";
+    repo = "hikari";
+    rev = version;
+    sha256 = "sha256-4uMY0NZaLxl52Wl471xQ6JqWWVW5QzMOyrXeVaZltfo=";
+    leaveDotGit = true;
+  };
+
+  patchInputs = [
+    git
+  ];
+
+  propagatedBuildInputs = [
+    aiohttp
+    attrs
+    multidict
+    colorlog
+  ] ++ lib.optional enableRestServer [ pynacl ];
```

Comment:
Would you please add this dependency as `passthru.optional-dependencies`?
see, https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#optional-extra-dependencies-python-optional-dependencies

Reply:
Fixed.